recipes/libffi/all/conanfile.py

```diff
@@ -35,6 +35,10 @@ def _settings_build(self):
         # TODO: Remove for Conan v2
         return getattr(self, "settings_build", self.settings)
 
+    @property
+    def _is_msvc_like(self):
+        return is_msvc(self) or (self.settings.compiler == 'clang' and self.settings.get_safe('compiler.runtime_version') is not None)
```

Comment:
I guess `compiler.runtime` is more reliable, a user may not set `compiler.runtime_version`

Reply:
That's not what I see in conan client source code.

Reply:
https://github.com/conan-io/conan/blob/develop/conans/test/functional/toolchains/cmake/test_cmake_toolchain_win_clang.py

Reply:
They are also based around compiler.runtime, I'm not sure it's a proof that runtime_version is mandatory.